substratevm/DEBUGINFO.md

```diff
@@ -174,37 +174,41 @@ configure extra source search paths (see below).
 Configuring source paths in gdb
 -------------------------------
 
-In order for gdb to be able to locate the source files for your app
-classes, Graal classes and JDK runtime classes you need to provide gdb
-with a list of source root dirs using the 'set directories' command:
+By default gdb will look into `/path/to/sources/{jdk,graal,src}` directories to
```

Comment:
I think this would be clearer:

    By default will look employ the three local directory roots `sources/{jdk,graal,src}` to locate the source files for your app classes, Graal classes and JDK runtime classes. If the sources cache is not located in the directory in which you run gdb you can configure the required paths using the following command:

Reply:
This should now be resolved